src/services/send-sqs-message.ts

```diff
@@ -0,0 +1,16 @@
+import { SQSClient, SendMessageRequest, SendMessageCommand, SendMessageCommandOutput } from '@aws-sdk/client-sqs';
+import assert from 'node:assert/strict';
+
+export const sendSqsMessage = async (
+  messageBody: string,
+  queueUrl: string | undefined,
+): Promise<SendMessageCommandOutput> => {
+  assert(process.env['AWS_REGION'], 'AWS_REGION environment variable not set');
```

Comment:
think we had issues before where asserts werent showing in the aws logs so we couldn't figure out that env vars were missing. could we replace this with the logger?